build/jobs/run-tests-aks.yml

```diff
@@ -32,7 +32,8 @@ jobs:
             $environmentVariableName = $secret.Name.Replace("--","_")
 
             $secretValue = Get-AzKeyVaultSecret -VaultName $(DeploymentEnvironmentName)-ts -Name $secret.Name
-            Write-Host "##vso[task.setvariable variable=$($environmentVariableName)]$($secretValue.SecretValueText)"
+            $plainValue = ([System.Net.NetworkCredential]::new("", $secretValue.SecretValue).Password).ToString()
```

Comment:
>$plainValue [](start = 12, length = 11)

Maybe it's worth to create second PR with `-AsPlainText` to see is it working or not. If it's not working we can close it, and if does, I would prefer to use offical way rather than this hack.

Reply:
Marius has confirmed that the Az module running right now is 5.0, and that variable isn't available until 5.3. 5.0 was released end of October and 5.3 end of December, so we could make the change in about 2 months if that is a consistent timeframe.

Reply:
I would add comment about this magic conversion and why we don't use `-AsPlainText`. It would probably stop working in two years, and people would scratch their heads about this line and what is going on here and why it's not using plaintext flag. 
Not insisting, tho.